skaffold.yaml

```diff
@@ -16,18 +16,21 @@ build:
         - harbor.toolchain.lead.prod.liatr.io/liatrio/springtrader-builder:latest
         - harbor.toolchain.lead.prod.liatr.io/liatrio/springtrader-runner:latest
   local: {}
-### [build] 
+### [build]
 
-### [deploy] 
+### [deploy]
 deploy:
   helm:
+    flags:
+      upgrade: ["--timeout", "600s"]
+      install: ["--timeout", "600s"]
```

Comment:
These flags should only be on solution2 branch

Reply:
There's a conflict when trying to merge solution2 if these fields don't already exist. I ran through labs 1-2 with these flags and everything worked fine for me. Is there are reason these timeout flags shouldn't be there?

Reply:
these shouldn't be there in solution1 because we aren't using `skaffold deploy` in lab 1, only lab 2 in CI.  the lab 2 instructions say to add these lines to the skaffold.yaml file: https://ignitelab.liatr.io/#/2-continuous-delivery/lab/skaffold-istio